public/javascripts/SVLabel/src/SVLabel/modal/ModalMission.js

```diff
@@ -110,7 +110,8 @@ function ModalMission (missionContainer, neighborhoodContainer, uiModalMission,
                 distanceString;
                 templateHTML = distanceMissionHTML;
 
-            if(missionContainer.isTheFirstMission()){
+            if ((missionContainer.onlyMissionOnboardingDone() || missionContainer.isTheFirstMission())
+                && !svl.storage.get('completedMissionAnonymously')) {
```

Comment:
Changed, I added the storage value to both `isTheFirstMission()` and `onlyMissionOnboardingDone` so that the same logic as in the highlighted if statement applies.